### PR TITLE
Build middle section - Works & About me sections

### PR DESCRIPTION
**Pull request by dicodiaz**
I built the next 2 sections of the website taking into account the following requirements:

- The mobile website should include:
  - "Works" section
  - The "About myself" section (placed after the projects and before the contact form).
- In order to layout the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to layout an individual card.
- Please use ul and li for categories in the "About myself" section. Languages, Frameworks, and Skills should be list items.
- You don't need to implement the detail modal window that requires JavaScript.
- You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma.
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,10 +43,149 @@ <h3 class="box subtitle">I am a Software Developer</h3>
...
+            </p>
+            <ul class="box2 list2">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <div class="box2">
+                <button type="button" class="button2">See Project</button>
+            </div>
```
Comment (carlos-ssh) [inline index.html]: - Kindly fix the style of your section:
Screen Shot 2021-08-13 at 19 26 40

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @dicodiaz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Your project it's very nice! i like it! ... just fix that issue with the style and request another code review.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dicodiaz** [comment]
Hello Carlos, thank you very much for your review! Yeah, this was a silly mistake and I didnt't notice it changed when I modified the README. I do have one question: Is the rest of the code OK? Does that mean I only have to change this and the project would be approved? Please let me know if I have any more mistakes/pending changes so I don't waste another review request only in this silly mistake.

File: css/styles.css [hunk 2]
```diff
@@ -162,3 +182,224 @@ button:disabled {
...
+  justify-content: center;
+}
+
+.project {
+  /* background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 61.94%
+    ); */
```
Comment (safafa) [inline css/styles.css]: - [ ] [OPTIONAL] Kindly remove all the commented code. To increase your code readablility

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @dicodiaz  great job working on the requested changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**